cluster-autoscaler/estimator/estimator.go

```diff
@@ -72,3 +73,9 @@ type EstimationLimiter interface {
 	// just not expected to add any more nodes.
 	PermissionToAddNode() bool
 }
+
+// EstimationPodOrderer is an interface used to determine the order of the pods
+// used while binpacking during scale up estimation
+type EstimationPodOrderer interface {
+	Process(pods []*apiv1.Pod, nodeTemplate *framework.NodeInfo, nodeGroup cloudprovider.NodeGroup) []*apiv1.Pod
```

Comment:
nit: Maybe "Order" would be a more accurate name? Process is pretty generic.

Reply:
Done.